zer0-os/zOS
# feat(message): add download image placeholder error state

### What does this do?
- adds download image placeholder error state

### Why are we making this change?
- improved ui/ux

### How do I test this?
- run tests as usual
- run UI and check error state for image downloading on a message

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?

<img width="1080" alt="Screenshot 2024-10-08 at 07 17 41" src="https://github.com/user-attachments/assets/ed852516-ca72-40bf-b209-a1135273d4b6">
